pkg/agent/hooks/conn/factory.go

```diff
@@ -52,9 +52,8 @@ func (factory *Factory) ProcessActiveTrackers(ctx context.Context, t chan *model
 		case <-ctx.Done():
 			return
 		default:
-			ok, requestBuf, responseBuf, reqTimestampTest, resTimestampTest := tracker.IsComplete()
+			ok, requestBuf, responseBuf, reqTimestampTest, resTimestampTest, clientID := tracker.IsComplete()
```

Comment:
Please refactor `tracker.IsComplete()` function, it should not return these many parameters.